spec/titles/bf:origin_date_from_240_subfield_f.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'linkeddata'
+
+# X30, 240, etc. - Uniform titles - R1
+
+describe 'bf:originDate from uniform title' do
```

Comment:
I would be specific with these descriptions.  In this case, it is from the 240 subfield f.